tests/unit_tests/agents/conversational_chat/test_output_parser.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from langchain.agents import AgentType, initialize_agent
+from langchain.chat_models import ChatOpenAI
+from langchain.memory import ConversationBufferWindowMemory
+
+# set no tools available (generate only)
```

Comment:
Encapsulate the test within a function. The test needs to test the agent output parser, so it should not instantiate anything but the parser. 

This test can rely heavily on @pytest.mark.parameterize to test the various parsing scenarios.

Reply:
I'm on the go right now, but will try to provide a snippet if I manage to do it before I have to turn my laptop off :)

Reply:
I worked with ChatGPT - we rewrote the tester and made sure it passes ruff and mypy checks.